### ISSUE TITLE
Tabs and TabsContent scenario

### ISSUE THREAD

**Issue by gmiseria** [stsrki/Blazorise#773]
In the scenario with tabs and tabscontent the SelectedTabChanged event is called twice. The first time with the new selected tabname and the second time with the previous tabname. As result the tab does not change.

In the scenario with tabs and content the SelectedTabChanged event is called only once

**Comment by stsrki** [comment]
Hi, can you post an example so I can reproduce it?

**Comment by gmiseria** [comment]
Hi, in the example below I reproduced the twice event call the first with the right tabname and the second with the previous tabname
I observed that if I use the tabs and tabscontent without the card component the event is called only once.
```
<Card Margin="Margin.Is1">
    <CardHeader Margin="Margin.Is1">
        <CardTitle Size="5">
            Commessa:Test
        </CardTitle>
        <Tabs SelectedTab="@selectedTabCommessa" SelectedTabChanged="@OnSelectedTab">
            <Items>
                <Tab Name="anag">Anagrafica</Tab>
                <Tab Name="risorse">Risorse</Tab>
                <Tab Name="fatture">Ordini / Fatture</Tab>
                <Tab Name="result">Risultati</Tab>
            </Items>
        </Tabs>
    </CardHeader>

    
    <CardBody Margin="Margin.Is0">
        <TabsContent @ref="tabsContent">
            <TabPanel Name="anag">
                Panel 1
            </TabPanel>
            <TabPanel Name="risorse">
                Panel 2

            </TabPanel>
            <TabPanel Name="fatture">
                Panel 3
            </TabPanel>
            <TabPanel Name="result">
                Panel 4
            </TabPanel>
        </TabsContent>
    </CardBody>
    <CardFooter>
        <button type="button" class="btn btn-secondary mr-auto" >Cancel</button>
        <button type="button" class="btn btn-success ml-auto" >Save</button>
    </CardFooter>
</Card>
```

```

@code{
    private TabsContent tabsContent;
    private string selectedTabCommessa { get; set; } = "anag";
    private void OnSelectedTab(string nameTab)
    {
        Console.WriteLine($"Tab Selected {nameTab},{selectedTabCommessa}");

        selectedTabCommessa = nameTab;
        tabsContent.SelectPanel(nameTab);


    }

}

```

**Comment by stsrki** [comment]
Tested again and it seems the issue was fixed by some of the changes I did in 0.9.3.

Closing now.

**Comment by Baggs** [comment]
I had similar behavior with event being called twice and selection returing to original tab. But in my case I forgot to set current tab in OnSelectedTab. In above's example it's:

> selectedTabCommessa = nameTab;

After adding this assignment everything works fine :)
